### PR TITLE
hw3-chanuk

### PR DESCRIPTION
**Pull request by HeoChanUk**
hw3-chanuk

### CODE REVIEW

File: class02/homework/heochanuk/hw3/hw03_README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+./splitted_dataset/train/<class#>: 4​
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S| 	1.0 	|	|0:01:18.817406 	|64 	|0.00355 	|----
+|EfficientNet-B0| 	1.0 	|	|0:00:18.494986 	|64 	|0.00245 	|----
+|DeiT-Tiny 	|1.0 	|	|0:00:16.258080 	|64 	|5e-05 	|----
+|MobileNet-V3-large-1x 	|1.0 	|	|0:00:13.059633 	|64 	|2.900e-03 	|----
```
Comment (JerryJLee) [inline class02/homework/heochanuk/hw3/hw03_README.md:23]: FPS 측정 방법이랑.. 모델 파일들 (*.xml, *.bin 각각 4개씩)은 커밋에 왜 누락되었는지..
